roc/roc.go

```diff
@@ -1,3 +1,169 @@
+// WARNING: This file has automatically been generated on Sun, 20 Oct 2019 18:32:28 CEST.
+// Code generated by https://git.io/c-for-go. DO NOT EDIT.
+
 package roc
 
-// TODO: replace this with generated bindings
+/*
+#cgo CFLAGS: -I"../include"
+#cgo LDFLAGS: -lroc
+#include "../include/roc/address.h"
+#include "../include/roc/config.h"
+#include "../include/roc/context.h"
+#include "../include/roc/frame.h"
+#include "../include/roc/log.h"
+#include "../include/roc/platform.h"
+#include "../include/roc/receiver.h"
+#include "../include/roc/sender.h"
+#include <stdlib.h>
+#include "cgo_helpers.h"
+*/
+import "C"
+import "unsafe"
+
+// AddressInit function as declared in roc/address.h:86
+func AddressInit(address *Address, family Family, ip string, port int32) int32 {
```

Comment:
Can we unexport all these functions (for which we're creating exported wrappers)?

Reply:
yup, still working on it,

Reply:
somehow, even if I tell c-for-go to unexport functions, it still has them exported :( Fixed manually

Reply:
I've created #7.